src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useMultiHopTradeStepAllowance.tsx

```diff
@@ -0,0 +1,29 @@
+import { type AccountId, fromAccountId } from '@shapeshiftoss/caip'
+import type { TradeQuoteStep } from '@shapeshiftoss/swapper'
+import { useMemo } from 'react'
+import { useAllowance } from 'react-queries/hooks/useAllowance'
+
+export function useMultiHopTradeStepAllowance(
```

Comment:
where did this come from, not used anywhere?

Reply:
Nice catch, this sneaked in in this diff - we now use `useAllowance()` since we don't do `adapter.getAddress()` business anymore! Removed in 2775b5fce0ed70ddec3f5d29f8bc9cfb8fcc5990